.github/workflows/test.yml

```diff
@@ -23,12 +23,24 @@ jobs:
     uses: ./.github/workflows/.reusable_test_image.yml
     needs: build
     secrets: inherit
+    with:
+      gateway-image: docker.io/kong/kong-dev:pr-14287-ubuntu # TODO: Revert this line before merging
```

Comment:
Remember to revert this.

Reply:
Reverted. Let's wait for the outcome of the CI.